core/sqf/export/limited-support-tools/QPDrawer/PlanForest

```diff
@@ -0,0 +1,854 @@
+        AddParallelism Node:=Node, bTop:=True
+        AddParallelism Node:=Node, bTop:=False
+    End If
+    If Op(Node).iLeft > 0 Then                              ' I have a left child
+        AddOperator Node:=Op(Node).iLeft, Level:=Level + 1
+    End If
+    If Op(Node).iRight > 0 Then                             ' I have a right child
+        AddOperator Node:=Op(Node).iRight, Level:=Level + 1
+    End If
+End Sub ' AddOperator
+Sub AlignParents(ByVal Lineage As Integer)                  ' Align all ancestors
+    Dim Parent As Integer
+    Dim M As Single
+    Dim Dad As Shape
+    Dim LC As Shape
+    Dim RC As Shape
+    
+    Do
+        Parent = Op(Lineage).iParent
+        If Parent > 0 Then
+            Set Dad = Tablet.Shapes(Op(Parent).sOpShape)
+            Set LC = Tablet.Shapes(Op(Op(Parent).iLeft).sOpShape)
+            If Lineage = Op(Parent).iRight Then
+                Set RC = Tablet.Shapes(Op(Lineage).sOpShape)
+                M = (LC.Left + RC.Left + ((LC.Width + RC.Width) / 2)) / 2
+            Else
+                M = LC.Left + (LC.Width / 2)
+            End If
+            M = M - (Dad.Left + (Dad.Width / 2))
+            If M < 0 Then
+                M = WorksheetFunction.Max(M, -Dad.Left)
+            End If
+            Dad.IncrementLeft M
+            If Op(Parent).bParallel Then
+                Tablet.Shapes(Op(Parent).sTPShape).IncrementLeft M
+                Tablet.Shapes(Op(Parent).sBPShape).IncrementLeft M
+            End If
+            Indent(Op(Parent).iLevel) = Dad.Left + Dad.Width
+        End If
+        Lineage = Op(Lineage).iParent
+    Loop Until Lineage = 0
+End Sub ' AlignParents
+Function OpDescription(ByVal Node As Integer) As String     ' Build description text
+    Dim Desc As String
+    
+    Op(Node).bParallel = False                              ' Until proven otherwise
+    Desc = Op(Node).sName                                   ' 19Aug2005
+    If Node = UBound(Op) Then ' top of tree - identify stmt ' 19Aug2005
+        Desc = StmtName & " - " & Desc                      ' 19Aug2005
+    End If                                                  ' 19Aug2005
+    Desc = Format(Op(Node).dCard, "Scientific") & Chr(10) & _
+            Desc & " [" & Node & "]"                        ' 19Aug2005
+    Select Case Op(Node).sName
+        Case "cursor_delete", _
+             "cursor_update", _
+             "file_scan", _
+             "trafodion_scan", _
+             "file_scan_unique", _
+             "trafodion_load", _
+             "index_scan", _
+             "trafodion_index_scan", _
+             "trafodion_vsbb_scan", _
+             "trafodion_vsbb_upser", _
+             "index_scan_unique", _
+             "insert", _
+             "insert_vsbb", _
+             "subset_delete", _
+             "subset_update", _
+             "trafodion_update", _
+             "trafodion_delete", _
+             "trafodion_insert", _
+             "trafodion_vsbb_updat", _
```

Comment:
Actually, I don't think we need to handle UDF, tuple, tuplelist, because there is no table name associated with them, but if trafodion_merge is a leaf then we should probably add it. If missing, I think it will still display the operator, just not the table name it is operating on.